crates/bevy_time/src/time.rs

```diff
@@ -122,11 +126,26 @@ impl Time {
     }
 
     /// The time from startup to the last update in seconds
+    ///
+    /// If you need an f32 value, it is highly recommended to use [`Time::seconds_since_startup_f32_wrapped`]
+    /// instead of casting it. Casting to an f32 can cause floating point precision issues pretty fast.
     #[inline]
     pub fn seconds_since_startup(&self) -> f64 {
         self.seconds_since_startup
     }
 
+    /// The time from the last wrap period to the last update in seconds
+    ///
+    /// When used in shaders, the time is limited to f32 which can introduce floating point precision issues
+    /// fairly quickly if the app is left open for a while. This will wrap the value to 0.0 on the update after
+    /// the `Self::max_wrapping_period`
```

Comment:
```suggestion
    /// When used in shaders, the time is limited to `f32` which can introduce floating point precision issues
    /// fairly quickly if the app is left open for a while. 
    /// This will wrap the value to 0.0 on the update after the `Self::max_wrapping_period`.
```